xml/System.Net.Http/HttpClient.xml

```diff
@@ -28,6 +28,10 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+
+ > [!WARNING]
+ > <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. Instantiating an HttpClient class for every request will exhaust the number of sockets available under heavy loads. This will result in a <xref:System.Net.Sockets.SocketException>.
```

Comment:
```suggestion
 > - <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. Instantiating an <xref:System.Net.Http.HttpClient> class for every request will exhaust the number of sockets available under heavy loads and result in a <xref:System.Net.Sockets.SocketException>. However, having long-lived <xref:System.Net.Http.HttpClient> instances can lead to stale DNS usage and non-functional connections. Use one of the following options to avoid these problems:
>
> - Use static <xref:System.Net.Http.HttpClient> instances, which are recycled on a regular basis.
> - Use <xref:System.Net.Http.HttpClientFactoryExtensions.CreateClient(IHttpClientFactory)> to create the <xref:System.Net.Http.HttpClient>.
> - On .NET Core and .NET 5.0 and later, use static instances and set <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionLifetime>.
```